pkg/cache/application.go

```diff
@@ -71,6 +71,7 @@ func (app *Application) String() string {
 
 func NewApplication(appID, queueName, user string, groups []string, tags map[string]string, scheduler api.SchedulerAPI) *Application {
 	taskMap := make(map[string]*Task)
+	taskGroups := make(map[string]*TaskGroup)
```

Comment:
Does it have to be a pointer? Probably not.

Reply:
Thank you for your reply, I agree with you.

Reply:
> Does it have to be a pointer? Probably not.

Thank you for your reply, I agree with you.